lib/wikitrivia/game.ex

```diff
@@ -17,10 +20,26 @@ defmodule Wikitrivia.Game do
     Agent.update(agent_name_by_game_id(game_id), award_points(player_name, points))
   end
 
+  # Starts the pose question -> show results loop. Ends after num_questions_left reaches 0.
+  # game_timer_callback function will be called after every timed state change.
+  def start(game_id, game_timer_callback) do
+    start_question(game_id, game_timer_callback)
+  end
+
+  # State structure of a Game. Details:
+  #   * players - Set of all players in this game.
+  #   * scores - Map of all player scores in this game, keyed by player name
+  #   * num_questions_left - Number of questions remaining to answer in this game
+  #   * timer_state - Current state of timed component of this game. :off before entering timed
+  #       periods, otherwise the name of the period (:question, :question_results, :done)
```

Comment:
@denheck To be honest though, I'm not all that confident about the state selection here, so I'm sure this will change again.